week-2/B-boolean-literals/exercise.js

```diff
@@ -6,6 +6,9 @@
 */
 
 var codeYourFutureIsGreat = true;
+var mozafarIsCool = false;
+caluclationCorrect = true;
```

Comment:
Make sure to use `var` when declaring your variables. Otherwise they are [global](https://www.w3schools.com/js/js_scope.asp), which isn't a good thing :-).